src/main/webapp/WEB-INF/queue.xml

```diff
@@ -93,6 +93,14 @@
         <rate>1/s</rate>
         <bucket-size>1</bucket-size>
     </queue>
+    
+    <queue>
+        <name>put-comment-document-queue</name>
+        <!-- Configuration allows for 10 documents to be queued when the bucket is not full -->
```

Comment:
It seems that the other comments all reflect the rate (all except `send-email-queue`, which I think might be a mistake). Indeed, after [reading](https://cloud.google.com/appengine/docs/standard/java/config/queueref) about what the `bucket-size` option really means, it seems that [it controls the burstiness](https://en.wikipedia.org/wiki/Token_bucket), which is how much the rate is allowed to spike beyond the average. It does not affect the average, so it should be 5, not 10.